lib/osf-components/addon/components/institutions-list/template.hbs

```diff
@@ -24,5 +25,9 @@
                 {{/if}}
             </div>
         </list.item>
+        <list.empty>
+            {{t 'osf-components.institutions-widget.no_affiliated_institution.node'
+                nodeType=(if @node.isRegistration 'registration' 'project')}}
```

Comment:
These need to be translated too. Something like
```hbs
{{t (if @node.isRegistration
    'osf-components.institutions-widget.no_affiliated_institution.registration'
    'osf-components.institutions-widget.no_affiliated_institution.project'
)}}
```
or
```hbs
{{#if @node.isRegistration}}
    {{t 'osf-components.institutions-widget.no_affiliated_institution.registration'}}
{{else}}
    {{t 'osf-components.institutions-widget.no_affiliated_institution.project'}}
{{/if}}
```
might be better.

(those are some long and weird keys... looking forward to when we eventually revisit our i18n approach)

Reply:
right. It gets long and ugly pretty quick. Are we still thinking about switching to `ember-intl`?

Reply:
Yup. We kind of have to. The analysis is done, we just have to fit it in.

Reply:
Yup. We kind of have to. The analysis is done, we just have to fit it in.